### PR TITLE
Update according Truffle team

### PR DESCRIPTION
**Pull request by solangegueiros**
## What

- Add more explanations in readme
- Add links to RSK, tutorial rsk-next-box and gitter
- Update method for get gas price

## Why

- Truffle team approved other tutorial in this style
- To be more friendly
- To link to the tutorial in DevPortal

## Refs

- [Trello card] (https://trello.com/c/YYay3DI4)

### CODE REVIEW

File: truffle-config.js [hunk 1]
```diff
@@ -1,26 +1,73 @@
...
     mainnet: {
       provider: () => new HDWalletProvider(mnemonic, 'https://public-node.rsk.co', 0, 1, true, "m/44'/137'/0'/0/"),
       network_id: 30,
-      gasPrice: 0x387EE40
-    },    
-    testnet: {
-      provider: () => new HDWalletProvider(mnemonic, 'https://public-node.testnet.rsk.co', 0, 1, true, "m/44'/37310'/0'/0/"),
-      network_id: 31,
-      gasPrice: 0x387EE40
+      gasPrice: Math.floor(minimumGasPriceMainnet * 1.02),
```
Comment (jonathansmirnoff) [inline truffle-config.js:61]: I like this!

File: truffle-config.js [hunk 2]
```diff
@@ -1,26 +1,73 @@
...
+    --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["latest",false],"id":1}' \
+    > .minimum-gas-price-testnet.json
+*/
+const gasPriceTestnetRaw = fs.readFileSync(".minimum-gas-price-testnet.json").toString().trim();
+const minimumGasPriceTestnet = parseInt(JSON.parse(gasPriceTestnetRaw).result.minimumGasPrice, 16);
+if (typeof minimumGasPriceTestnet !== 'number' || isNaN(minimumGasPriceTestnet)) {
+  throw new Error('unable to retrieve network gas price from .gas-price-testnet.json');
+}
+console.log("Minimum gas price Testnet: " + minimumGasPriceTestnet);
+
```
Comment (jonathansmirnoff) [inline truffle-config.js:20]: Great idea!

**Review by jonathansmirnoff — APPROVED** [review]
LGTM

File: truffle-config.js (hunk 1 above)
Comment (solangegueiros) [reply truffle-config.js:61]: Me too!
This was Brendan idea :)

File: truffle-config.js (hunk 2 above)
Comment (solangegueiros) [reply truffle-config.js:20]: This was Brendan idea too :)

Comment (solangegueiros) [reply truffle-config.js:20]: We are using in other tutorials, I will update all Truffle boxes to use it

**Review by owans — APPROVED** [review]
LGTM!

**Pull request merged**